### PR TITLE
✨ add Error Page

### LINKED ISSUE #29: 💅 style error page like designs
- add designs
- change copy based on error type

### PR DESCRIPTION
**Pull request by astroash**
- change copy based on params
- add default styling to buttons

fixes #29

NOTE: briefly looked into testing but it is complicated as the component relies on url params. As this component is not vital business logic it doesn't seem worth spending time on.

### CODE REVIEW

File: dashboard-app/src/Error.tsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const H1 = styled(_H1)`
   margin-bottom: ${SpacingEnum.medium};
 `;
 
-const H3Padded = styled(H3)`
+const H3 = styled(_H3)`
```
Comment (astroash) [inline dashboard-app/src/Error.tsx]: This changes the spacing between elements. I've setup the `SpacingEnum` to follow the design principle of spaces being multiples of each other.

Comment (eliascodes) [reply dashboard-app/src/Error.tsx]: I just changed the name of the variable, the spacing is the same as what you set it as.

Comment (astroash) [reply dashboard-app/src/Error.tsx]: There are 2 H3 elements. Previously only one had padding attached to it, now both do.

Comment (eliascodes) [reply dashboard-app/src/Error.tsx]: Oops, my bad. Will fix.

**Pull request merged**